src/components/layout/Header.tsx

```diff
@@ -24,7 +24,7 @@ function Header() {
   };
 
   let pfp = (
-    <Link to="/account/signin">
+    <Link to="/settings/profile/">
```

Comment:
My opinion is that it should take you to the sign in page when not signed in, but I'm okay with this change too